rodan-main/code/rodan/jobs/heuristic_pitch_finding/PitchFinding.py

```diff
@@ -479,16 +491,14 @@ def __glyph_type(g):
 
                 # find octave
                 if my_strt_pos <= clef_line:
-                    octave = 3 + int((clef_line - my_strt_pos + noteShift) / len(SCALE))
+                    octave = 3 + int((clef_line - my_strt_pos + noteShift) // len(SCALE))
```

Comment:
Replace all of the //'s with explicit floor functions - also add a comment describing what exactly this math is doing, so whoever sees this code next does not have to spend the time figuring out what's going on (like you did!)

Reply:
fixed in 48df766fcd4db298d81115d6a6199c12d207dd72.